website/content/docs/concepts/client-count.mdx

```diff
@@ -188,6 +193,33 @@ To reduce the number of non-entity tokens in use, consider switching to an authe
 [AppRole](/docs/auth/approle) instead of handing out directly-created tokens. Ensure that entities and
 [entity aliases](/api/secret/identity/entity-alias) exist for all login methods used to create batch tokens.
 
+### Tracking non-entity tokens
+
+As of Vault 1.9, non-entity tokens are tracked as unique clients based on the policies the token
+has access to and the namespace in which it was created. Non-entity tokens that are assigned 
+the same set of policies and are created in the same namespace will be tracked as the same client. 
+Conversely, if two non-entity tokens have a different policy set or are created in different namespaces
+they will be tracked as two separate clients. 
+
+Please note that before the release of Vault 1.9, non-entity tokens were each tracked separately. That is 
+to say, two non-entity tokens would always be counted as two separate clients.
+
+## Auditing clients
+
+As of Vault 1.9, the Vault Audit Log will contain a `client_id` field in the request. The `client_id` will
+be empty for requests that are untracked by the Audit Log. Such requests are unauthenticated requests / 
```

Comment:
This is confusing: if the request isn't tracked by the audit log, why do we need to talk about its client_id field being blank, why should I (the reader) care?

Reply:
Yeah, I should reword this -- basically, if a request uses a root token or a wrapped token, the client_id (a non-entity token which won't get a client_id and be tracked in the activity log, the client_id field in the request will be "". Initially I was thinking it might be good to let the reader know the cases in which they'll see an empty client_id field. Do you think it's worth adding something in about that?